src/igvfd/schemas/prediction.json

```diff
@@ -0,0 +1,70 @@
+{
+    "title": "Prediction",
+    "description": "Schema for submitting a file set of prediction files generated by CAMP.",
```

Comment:
Not sure we want to use abbreviations like CAMP. 

Maybe just "set of computational predictions"?